doc/hands_on/hands_on_dynamodb_vs.md

```diff
@@ -0,0 +1,229 @@
+In this guide we will use the docker version.
+You can however also use a different method.
+
+Steps for setup:
+ 
+1. Install DynamoDB:
+    ```shell script
+    docker run -p 8000:8000 amazon/dynamodb-local -jar DynamoDBLocal.jar -sharedDb -dbPath .
+    ``` 
+1. [Install AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+1. create `~/.aws/credentials` and fill in:
+   ```
+   [default]
+       aws_access_key_id = fakeMyKeyId
+       aws_secret_access_key = fakeSecretAccessKey
+       region = eu-central-1
+   ```
+   *The local DynamoDB does not check credentials. So we just need to add something for passing the checks in the CLI.*
+   
+In the following steps you have to append `aws help --endpoint-url http://localhost:8000` to all `aws dynamodb` commands. 
+
+
+## Setup sample data
+
+Now we are going to load some example data into the DynamoDB.
+For that we first of all need to create a DynamoDB table.
+We will do that suing the AWS CLI.
+For DynamoDB on AWS there is also a GUI.
+
+You can create the table by simply running the following command in a shell.
+
+``` shell script
+aws dynamodb create-table \
+    --table-name Books \
+    --attribute-definitions \
+        AttributeName=Title,AttributeType=S \
+    --key-schema \
+        AttributeName=Title,KeyType=HASH \
+    --provisioned-throughput \
+        ReadCapacityUnits=1,WriteCapacityUnits=1
+```
+
+You can verify that the table was created by running:
+
+```shell script
+aws dynamodb list-tables
+```
+
+The output should contain the `Books` table, we just created.
+
+### Load example data
+
+Now we will load the example data into the newly created table.
+
+Steps:
+
+1. [Download example data](./exampleData.json)
+2. Load the data into DynamoDB by running:
+
+``` shell script
+aws dynamodb batch-write-item --request-items file://./exampleData.json
+```  
+
+## Setup an Exasol database
+Now we need an Exasol database. In this guide we will use a local Exasol VM.
+Basically you can however select choose between the following options:
+
+* **Local Exasol VM (recommended)**
+    * free
+    * simple
+* [Exasol docker-db](https://github.com/exasol/docker-db)
+    * free
+    * only runs on linux
+* [Exasol public demo](https://docs.exasol.com/get_started/publicdemo/publicdemosystem.htm)
+    *  Only applicable with DynamoDB on AWS
+* Run [Exasol in the Cloud](https://docs.exasol.com/cloud_platforms/aws/cloud_wizard.htm)
+    * causes costs
+    * Only applicable with DynamoDB on AWS
+    
+Independent of which setup you choose it is important that the Exasol database can read the DynamoDB over network.
+Hence you can not use an Exasol DB running in the cloud in combination with an local DynamoDB (ok, it would be possible if you can open a port on your firewall, but probably you don't want to do so). 
```

Comment:
```suggestion
Hence you can not use an Exasol DB running in the cloud in combination with a local DynamoDB (ok, it would be possible if you can open a port on your firewall, but probably you don't want to do so). 
```